auslib/admin/views/rules.py

```diff
@@ -298,14 +299,21 @@ def __init__(self):
 
     @requirelogin
     def _post(self, transaction, changed_by):
-        if request.json and request.json.get("data_version"):
-            form = ScheduledChangeExistingRuleForm()
-        else:
-            form = ScheduledChangeNewRuleForm()
+        if request.json.get("change_type") != "delete":
+            if request.json and request.json.get("change_type") == "update":
+                form = ScheduledChangeExistingRuleForm()
+            elif request.json.get("change_type") == "new":
```

Comment:
Nit: "insert" would be better than "new", since we're naming the other two (update/delete) like database operations.